### PR TITLE
Added a new task, Add Clues !! :D

### PR DESCRIPTION
**Pull request by Beta-King**
### When you change [things like this in brackets] please remember to delete brackets after you change it.

# This pull request is related to issue(s), 

# The problem(s) I want to solve or The facilitie(s) I want to improve is/are,
[Mention the bug/facilitie solved/improved]

### My steps to solve it,
[Mention the files changed. Add what changes you have done.] 

### Screenshots,
[If any]

### Links

http://rawgit.com/Beta-King/labyrinth/master/index.html

### CODE REVIEW

**Comment by Beta-King** [comment]
Please review and Merge :D

File: tasks/AddClues.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Labyrinth: Add clues to lead to the various parts of the labyrinth !!
+
+The [FOSSASIA Labyrinth](https://github.com/fossasia/labyrinth/) allows you to contribute parts to a huge labyrinth.
+To make the game more interesting, please add a functionality to add music when the user enters a tile,
```
Comment (niccokunzmann) [inline tasks/AddClues.md]: this does not reflect the title in my eyes

**Comment by niccokunzmann** [comment]
@Beta-King Please get review from at least two other participants by posting in the chat.

**Comment by Beta-King** [comment]
Yeah i have done that :D

**Comment by Beta-King** [comment]
What are your thoughts for this task though ??

**Comment by niccokunzmann** [comment]
Thanks for asking:
- [Context GCI]: I think, this task is way too simple. It would be a beginner task, maybe.
- [What I like about the game]: is walking around, not having alerts pop up. I would like to remove many of them - that is personal and will not direct my maintenance and merging. I fear new ones will be added and the nice tiles of the players are hidden by alerts.

**Comment by Beta-King** [comment]
Thats interesting; i have a completely different approve to this game ;D Cause i think the additions of riddles and other clues will give an investigative spice to the game and even an alternative purpose to it :D

**Comment by niccokunzmann** [comment]
Maybe I misunderstood and it is not all about alert boxes. Maybe it is about adding riddles like the sphinx of math or logic puzzles. I do not know The task does not state this or I just did not read it carefully.
I also think, feedback from other people can be valuable, then, they would like to claim the task more.

**Comment by Beta-King** [comment]
Ohk, ill give a summary to the task :D @niccokunzmann

File: tasks/AddClues.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Requirements:
+- In the `tiles.js` file edit the `alterInfo` tag of various tiles to lead the user to the various parts of the labyrinth (e.g. new year tile, etc)
+- Note: the clues you give are logical and tangible
```
Comment (niccokunzmann) [inline tasks/AddClues.md]: I do not know what logical and tangible means. If you are a mentor and you have a disagreement with a student about whether this is tangible and logical, how do you both get to an agreement?

Comment (Beta-King) [reply tasks/AddClues.md]: hmm wait ill make some changes...

Comment (niccokunzmann) [reply tasks/AddClues.md]: Maybe you can give examples 3+ and from there we abstact.

Comment (niccokunzmann) [reply tasks/AddClues.md]: examples will also guide the people who claim.

**Comment by Beta-King** [comment]
Ohk i've made some small changes to summarise the task and its necessity @niccokunzmann

**Comment by Beta-King** [comment]
Thanks @Valley-valley

**Comment by arjunpandey2003** [comment]
@Beta-King I agree with @niccokunzmann and had written the same in your earlier PRs which were the same. You can instead also Update the Create-a-tile.md for the same giving the users a suggestion I think that would be better.

**Comment by Beta-King** [comment]
I guess the Create-a-tile.md is a task which falls under a single tile where as this task is telling the participants to create a trail of `alertInfo` messages over a number of tiles and hence i don't think that this will fall under its jurisdiction :D

**Comment by Beta-King** [comment]
Hey @niccokunzmann i've made a new task which i think is better and more difficult than this one; if this is not appealing enough should i add the other task instead ??

**Comment by Beta-King** [comment]
otherwise i can also add the other task in another PR :D

**Comment by arjunpandey2003** [comment]
Ok @Beta-King, it would be better if you could give a preview by enabling it.

**Comment by Beta-King** [comment]
enabling it how ??

**Comment by Beta-King** [comment]
@niccokunzmann --> three reviews approved, can we merge now :D

**Comment by arjunpandey2003** [comment]
@Beta-King By adding alerts of riddles and crosswords in a tile or making API for alerts like that.

**Comment by Beta-King** [comment]
@arjunpandey2003 --> thats the task right, do you want me give an example ??

**Comment by Beta-King** [comment]
APIs in this case is not required as its just a mere one line of code change per tile

**Comment by Beta-King** [comment]
Tile one: Follow the Sun (East//Left) until you see something sad (myYellowTile)
Tile two: Wait for the sun to set and as it is setting follow it again (West//Right) until the find what you desire 
Tile three: Horray you found something interesting :D

**Comment by Beta-King** [comment]
Something like this i guess :D

**Comment by Beta-King** [comment]
Can we merge now ?? :D

**Comment by Beta-King** [comment]
Thank you @sayak119 :D

**Comment by Beta-King** [comment]
@niccokunzmann, why did my travis fail ??

**Comment by Beta-King** [comment]
Why did my travis fail @niccokunzmann ??

**Comment by Beta-King** [comment]
what ??

**Comment by Beta-King** [comment]
wait im making a new PR !!

**Pull request closed without merging**